src/pages/dashboardUserEdit/EditUserRegistryPage.tsx

```diff
@@ -5,10 +5,11 @@ import { resolvePathVariables } from '@pagopa/selfcare-common-frontend/utils/rou
 import { useTranslation } from 'react-i18next';
 import ProductNavigationBar from '../../components/ProductNavigationBar';
 import { DASHBOARD_USERS_ROUTES } from '../../routes';
-import withUserDetail, { withUserDetailProps } from '../../decorators/withUserDetail';
+// import withUserDetail, { withUserDetailProps } from '../../decorators/withUserDetail';
+import withUserRegistry, { withUserRegistryProps } from '../../decorators/withUserRegistry';
 import EditUserRegistryForm from './components/EditUserRegistryForm';
 
-type Props = withUserDetailProps;
+type Props = withUserRegistryProps;
 
 function EditUserRegistryPage({ party, partyUser }: Props) {
```

Comment:
I suggested in the decorator a different name, please change it also here
```suggestion
function EditUserRegistryPage({ party, user }: Props) {
```